go.mod

```diff
@@ -3,11 +3,12 @@ module github.com/kanopy-platform/grafana-auth-proxy
 go 1.16
 
 require (
+	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869 // indirect
 	github.com/go-macaron/session v1.0.2 // indirect
 	github.com/go-sql-driver/mysql v1.6.0 // indirect
 	github.com/gosimple/slug v1.11.2 // indirect
 	github.com/grafana/grafana v6.1.6+incompatible
```

Comment:
This still imports the v6 grafana model definitions is that an issue for using the proxy against v8 grafana?

Reply:
I will update this particular PR to be entirely grafana 6 supported.  Then I will open a new PR for grafana 8